app/services/activitypub/fetch_remote_status_service.rb

```diff
@@ -80,4 +80,20 @@ def expected_activity_type?
   def expected_object_type?
     equals_or_includes_any?(@json['type'], ActivityPub::Activity::Create::SUPPORTED_TYPES + ActivityPub::Activity::Create::CONVERTED_TYPES)
   end
+
+  def fetch_status(uri, id_is_known, on_behalf_of = nil)
```

Comment:
simple little wrapper around `fetch_resource` to catch the error, as discussed.

I was surprised that this was written to not raise an error if there was a 404, seems like it should in order to trigger a retry? I kept that behavior, but we could now easily reraise here if we wanted to.

Reply:
We don't need to trigger a retry on 404 since there is basically no situation where we know a post URI before it goes live. A 404 should not cause a retry.